src/worldcereal/utils/map.py

```diff
@@ -1,51 +1,14 @@
 import copy
-from typing import List
 
 import geopandas as gpd
 import leafmap
-import matplotlib.pyplot as plt
-import numpy as np
-import rasterio
 from ipyleaflet import DrawControl, LayersControl, Map, SearchControl, basemaps
 from loguru import logger
 from openeo_gfmap import BoundingBoxExtent
 from shapely import geometry
 from shapely.geometry import Polygon, shape
 
-
-def get_probability_cmap():
-    colormap = plt.get_cmap("RdYlGn")
-    cmap = {}
-    for i in range(101):
-        cmap[i] = tuple((np.array(colormap(int(2.55 * i))) * 255).astype(int))
-    return cmap
-
-
-COLORMAP = {
-    "active-cropland": {
-        0: (232, 55, 39, 255),  # inactive
-        1: (77, 216, 39, 255),  # active
-    },
-    "temporary-crops": {
-        0: (186, 186, 186, 0),  # no cropland
-        1: (224, 24, 28, 200),  # cropland
-    },
-    "maize": {
-        0: (186, 186, 186, 255),  # other
-        1: (252, 207, 5, 255),  # maize
-    },
-    "croptype": {
-        1: (103, 60, 32, 255),  # barley
-        2: (252, 207, 5, 255),  # maize
-        3: (247, 185, 29, 255),  # millet_sorghum
-        4: (186, 186, 186, 0),  # other_crop
-        5: (167, 245, 66, 255),  # rapeseed
-        6: (85, 218, 218, 255),  # soy
-        7: (245, 66, 111, 255),  # sunflower
-        8: (186, 113, 53, 255),  # wheat
-    },
-    "probabilities": get_probability_cmap(),
-}
+LOCALTILESERVER_PORT = 8889
```

Comment:
what do we still need this for? Does this local tile server stuff work? I thought we were not going to try to go this far.

Reply:
moving product visualization to utils.py...